wwe-service-client-api/sample.html

```diff
@@ -0,0 +1,349 @@
+<!DOCTYPE html>
```

Comment:
I suggest to rename the file by index.html?

Reply:
Sure, will rename that file.